Class Presentations/reveal-presentations/SelfConf2016.html

```diff
@@ -0,0 +1,582 @@
+							<input type="checkbox" id="goal3" />
+							<label for="goal3">What Has Been Easy? Difficult? Fun?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal4" />
+							<label for="goal4">What Are Some Things We Didn't Think About?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal5" />
+							<label for="goal5">What Lessons Did We Learn?</label>
+						</li>
+					</ul>
+				</section>
+
+				<section>
+					<h2>Who Are We?</h2>
+				</section>
+
+				<section>
+					<h2>Brandy.</h2>
+					<img src="../presentation-images/selfconf2016/brandy.png" width="292px" height="405px">
+				</section>
+
+				<section>
+					<h2>Kyle.</h2>
+					<img src="../presentation-images/selfconf2016/kyle.jpg" width="292px" height="390px">
+				</section>
+
+				<section>
+					<h2>What Is This Project?</h2>
+				</section>
+
+				<section>
+				<p><strong>D-Code</strong> is an after-school program in which students
+					learn the fundamentals of coding &amp; software development.</p>
+				</section>
+
+				<section>
+				<p>The program is held at Detroit Edison Public School Academy (DEPSA)
+					Early College of Excellence, a charter high school just north of
+					Eastern Market.</p>
+					<img src="../presentation-images/selfconf2016/ece_triptych.png">
+				</section>
+
+				<section>
+				<p>We meet twice a week.</p>
+				</section>
+
+				<section>
+					<h2>Goals</h2>
+					<ul class="checklist">
+						<li>
+							<input type="checkbox" id="goal0" checked />
+							<label for="goal0">Who Are We, and What Is This Project?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal1" />
+							<label for="goal1">Why Did We Decide to Do This?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal2" />
+							<label for="goal2">What Resources Did We Use?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal3" />
+							<label for="goal3">What Has Been Easy? Difficult? Fun?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal4" />
+							<label for="goal4">What Are Some Things We Didn't Think About?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal5" />
+							<label for="goal5">What Lessons Did We Learn?</label>
+						</li>
+					</ul>
+				</section>
+
+
+
+
+
+
+
+
+				<section>
+					<h2>Why Did We Decide to Do This?</h2>
+				</section>
+
+				<section>
+				<h3>We were already interested in community outreach.</h3>
+				<aside class="notes"> We had been visiting city and metro area schools
+					to speak with groups of students about what we did as app developers.
+				</aside>
+				</section>
+
+				<section>
+				<h3>We saw a need.</h3>
+				<aside class="notes"> -students asked if DL had internships, answer was
+					no
+					-we could only point them to resources and hope they figured it out
+					-knew that their peers from wealthier backgrounds could access people,
+					AP courses, clubs, etc
```

Comment:
-Obvious lack of black technology role models available in the tech community at large but specifically in side the walls of local schools